docs/general/source/development_plan.rst

```diff
@@ -0,0 +1,225 @@
+.. _dev-plan:
+
+Development Plan
+================
+
+.. todo::
+
+    This is a first draft. Check completeness and consistency of all items. Check and approbriateness of  timelines for the release.
+
+.. todo::
+
+    Check and complete list of contributors for the various items
+
+
+The following sections provide an overview of various development targets for NWB:N and what they entail.
+
+.. _dev-nwb2beta:
+
+NWB:N 2.0 beta (11/10/2017)
+---------------------------
+
+This is a first public beta release of the new version NWB:N. The stability of all modules can't be ensured. Backward compatibility cannot be promised at this point yet. Further changes to the APIs as well as the format are still planed between this beta and the first full release of NWB 2.0 (see Section :ref:`dev-nwb2` for details). The intent of this beta release is to enable early adopters to start exploring the new format and software.
```

Comment:
`The stability of all modules can't be ensured.` what do you mean by this? should be more precise.

Reply:
s/planed/planned/